src/java/org/jivesoftware/openfire/session/LocalOutgoingServerSession.java

```diff
@@ -449,77 +433,15 @@ private static LocalOutgoingServerSession secureAndAuthenticate(String hostname,
             // Get new stream features
             features = reader.parseDocument().getRootElement();
             if (features != null) {
-                // Check if we can use stream compression
-                final Connection.CompressionPolicy compressionPolicy = connection.getConfiguration().getCompressionPolicy();
-                if (Connection.CompressionPolicy.optional == compressionPolicy) {

```

Comment:
It was never used in the first place. A couple of lines down, it reads: if
(zlibSupported) { zlibSupported = false; }

On 4 January 2016 at 23:14, Tom Evans <notifications@github.com> wrote:

> In
> src/java/org/jivesoftware/openfire/session/LocalOutgoingServerSession.java
> <https://github.com/igniterealtime/Openfire/pull/479#discussion_r48789078>
> :
>
> > @@ -449,77 +433,15 @@ private static LocalOutgoingServerSession secureAndAuthenticate(String hostname,
> >              // Get new stream features
> >              features = reader.parseDocument().getRootElement();
> >              if (features != null) {
> > -                // Check if we can use stream compression
> > -                final Connection.CompressionPolicy compressionPolicy = connection.getConfiguration().getCompressionPolicy();
> > -                if (Connection.CompressionPolicy.optional == compressionPolicy) {
>
> Are we dropping support for S2S compression?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/igniterealtime/Openfire/pull/479/files#r48789078>.
>